src/Filesystem/PRIVACY.md

```diff
@@ -0,0 +1,21 @@
+# File System Privacy
+Disclaimer: This documentation does not warrant completeness or correctness. Please report any missing or wrong information using the [ILIAS issue tracker](https://mantis.ilias.de) or contribute a fix via [Pull Request](../../docs/development/contributing.md#pull-request-to-the-repositories).
+
+## General Information
+The File System is a special case as it is managing the storage of files which are given to it and obtained by other Services and Modules. Therefore most of the privacy relevant parts take part in these services and components which use the File System. Currently this is the ILIAS Resource Storage Service (IRSS) as well as many others, but in the future more Modules and Services will be migrated from the direct usage of the File System to an indirect usage via the IRSS.
```

Comment:
Services and components read and write files using the Filesystem. 
Services and components which use the Filesystem might store, present, delete or export personal data. This is specified in their respective PRIVACY.md.